### DISCUSSION TITLE
Water scarcity estimation using CWatM

### DISCUSSION THREAD

**Discussion by LSRathore** [iiasa/CWatM#147] [Q&A]
Hello all,

I would like to estimate water scarcity using CWatM. The water scarcity index I am using is called CAD (cumulative abstraction to demand ratio), which is the ratio between total water withdrawal and water demand for all the sectors.

So, I would like to obtain water withdrawal (it should be only renewable water or actual abstraction) and water demand. It seems the total water demand is stored in the `self.var.totalWaterDemand`, I am confused which variables should be considered to calculate the total actual water withdrawal (only renewable). 

The CAD is generally calculated for surface and groundwater combined, but I would appreciate it if someone could direct me to the variables storing water withdrawal and demand separately for surface and groundwater.

In addition to CAD, there is one other water scarcity index WSI which is the ratio between water demand and availability. What variable could be used for total water availability in a grid? Does `channelStorage` mean the total river water in a grid?

Thank,
Lokendra

**Comment by dof1985** [comment]
Dear @LSRathore 

**Influencing the water mix - a simple approach**
The division of water demand to surface and groundwater at CWatM's most basic version can be made using the 'swAbstractionFrac' in the [WATER_DEMAND] section. 
One can define a positive fraction (e.g, 0.45), to set the share of water demand attempted to be satisfied by surface water (channel, reservoirs and lakes). The reminder would be withdrawn from renewable GW (limitAbstraction = TRUE) of renewable and fossil GW (limitAbstraction = FALSE; in [OPTIONS]).
Another option is to set a negative value to 'swAbstractionFrac', and provide average discharge ('averageDischargeInput') and average baseflow ('averageBaseflowInput') maps, so the share of surface water use is based upon the relative abundance of each water source.

**Relevant variables**
The withdrawal variables are the following:
Surface water -> act_SurfaceWaterAbstract (unit is Meter, needs to multiply by cellarea)
Groundwater -> nonFossilGroundwaterAbs (unit is Meter, needs to multiply by cellarea)
Total withdrawal -> act_totalWaterWithdrawal (this should be the sum of these two); (unit is Meter, needs to multiply by cellarea)

The demand -> totalWaterDemand
You can use the share of surface water withdrawal to divide the preferences for surface water vs. groundwater - but it would be highly uncertain, unless you have data to back up your settings.

**WSI**
To estimate water availability in global hydrological modelling one may use the runoff + water_inflow_from_upstream_cells (e.g., discharge converted to cubic meters). This would require post processing, and tackling some problems regarding shared waters (e.g., rivers crossing border back-and-forth, lakes that are shared between countries). Usually it is accepted to just use the runoff. (varible name is runoff, also in meters).

Hope it helps
Dor
